.github/workflows/build.yml

```diff
@@ -3,6 +3,7 @@ name: Build
 on:
   push:
   pull_request:
+  workflow_dispatch:
```

Comment:
Can you please exclude the changes from .github and .vscode from this PR? Happy to add them in a follow up but would prefer to not add in work outside of the PR scope.

Reply:
Fixed.